action.yml

```diff
@@ -6,6 +6,10 @@ inputs:
   directory:
     description: "The directory to scan"
     required: true
+  arguments:
+      description: "Passes through options"
```

Comment:
Change description to `"HTMLProofer options"`

Reply:
Done